go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/provenance-io/provenance
 
-go 1.15
+go 1.16
```

Comment:
Does wasmd v0.19.0 require go 1.16?  Are there any negative side effects of upgrading our go version here as well?

Reply:
I updated the wasmd from 1.15 to 1.16 for the v0.19.0-alpha tag.  Testing will show if there are any side effects and we will have to resolve them when they come up.  @iramiller mentioned that go 1.17 is supposed to be 5% faster, but we are holding off on upgrading to that.

Reply:
I have been using go 1.16 locally (including for building my local mainnet node) for several months now without any issues.